error: no matching function for call to 'min(double, float)'

When compiling the project without changing anything, I've got the following error:
```lib/grbl/src/nuts_bolts.cpp:161:79: error: no matching function for call to 'min(double, float)'```

I'm using VSCode with PlatformIO `v2.3.3`

Based on  https://github.com/RobTillaart/Arduino/issues/128 I was able to solve the error by changing `lib/grbl/src/nuts_bolts.cpp:161`

from
```c++
 limit_value = min((double)limit_value,fabs(max_value[idx]/unit_vec[idx]));
```

to
```c++
 limit_value = _min((double)limit_value,fabs(max_value[idx]/unit_vec[idx]));
```

--- a/lib/grbl/src/nuts_bolts.cpp
+++ b/lib/grbl/src/nuts_bolts.cpp
@@ -158,7 +158,7 @@ float limit_value_by_axis_maximum(float *max_value, float *unit_vec)
   float limit_value = SOME_LARGE_VALUE;
   for (idx=0; idx<N_AXIS; idx++) {
     if (unit_vec[idx] != 0) {  // Avoid divide by zero.
-      limit_value = min((double)limit_value,fabs(max_value[idx]/unit_vec[idx]));
+      limit_value = _min((double)limit_value,fabs(max_value[idx]/unit_vec[idx]));
     }
   }
   return(limit_value);

